core/service/src/engine/backup_operator.rs

```diff
@@ -129,16 +329,27 @@ where
     /// Observe that if secret sharing is used for backup (i.e. with a master key being shared with a set of custodians)
     /// then [`custodian_recovery`] _must_ be called first in order to ensure that the master key is restored,
     /// which is needed to allow decryption of the backup data.
```

Comment:
Should this be split into a separate component for exporting, allowing to use another export location than the backup?
Right now they are entangled and you can only use one. Alternatively should there be two different kinds of backup operators?